scripts/handle-user-data

```diff
@@ -33,6 +35,10 @@ mount_data_partition() {
     mkdir -p /mnt/$PARTNAME/var/lib/boot2docker/etc/docker
     ln -s /mnt/$PARTNAME/var/lib/boot2docker/etc/docker /etc/docker
 
+    mkdir -p /mnt/$PARTNAME/var/lib/origin
+    mkdir -p /var/lib/origin
+    mount --bind /mnt/$PARTNAME/var/lib/origin/ /var/lib/origin/
```

Comment:
@hferentschik bind mount actually keep copy of same file in 2 different locations, I am using it only for openshift case because symlink doesn't work currently. keep everything as bind mount means we are duplicating most of data and in docker case it will be images which are huge in size.

+1 on having a CI test which will do a image sanity test.